django/db/models/options.py

```diff
@@ -690,7 +690,7 @@ def get_path_to_parent(self, parent):
                 final_field = opts.parents[int_model]
                 targets = (final_field.remote_field.get_related_field(),)
                 opts = int_model._meta
-                path.append(PathInfo(final_field.model._meta, opts, targets, final_field, False, True))
+                path.append(PathInfo(final_field.model._meta, opts, targets, final_field, False, True, None))
```

Comment:
I think keyword arguments would make this more readable - it's not clear to the reader what the `False`, `True` and `None` mean without looking up the definition of `PathInfo`.